core/api/src/servers/graphql-main-server.ts

```diff
@@ -123,16 +123,18 @@ export async function startApolloServerForCoreSchema() {
   })
 }
 
-setupMongoConnection(true)
-  .then(async () => {
-    activateLndHealthCheck()
+if (require.main === module) {
+  setupMongoConnection(true)
+    .then(async () => {
+      activateLndHealthCheck()
 
-    const res = await bootstrap()
-    if (res instanceof Error) throw res
+      const res = await bootstrap()
+      if (res instanceof Error) throw res
 
-    await Promise.race([
-      startApolloServerForCoreSchema(),
-      startApolloServerForAdminSchema(),
-    ])
-  })
-  .catch((err) => baseLogger.error(err, "server error"))
+      await Promise.race([
+        startApolloServerForCoreSchema(),
+        startApolloServerForAdminSchema(),
+      ])
+    })
+    .catch((err) => baseLogger.error(err, "server error"))
+}
```

Comment:
we should move that to another file

Reply:
Done https://github.com/GaloyMoney/galoy/pull/3572/commits/7b806f1ccd581d03f639c3fe4d1af3a7cd53ef84